workitem/workitemtype_repository.go

```diff
@@ -181,6 +182,21 @@ func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UU
 	return &app.WorkItemTypeSingle{Data: &result}, nil
 }
 
+// List returns work item types that derives from PlannerItem type
+func (r *GormWorkItemTypeRepository) ListPlannerItems(ctx context.Context, spaceID uuid.UUID) ([]WorkItemType, error) {
+	var rows []WorkItemType
+	db := r.db.Select("id").Where("space_id = ? AND path::text LIKE '"+LtreeSafeID(SystemPlannerItem)+".%'", spaceID.String())
```

Comment:
@aslakknutsen @hectorj2f 
We have same function as if `LtreeSafeID` [here](https://sourcegraph.com/github.com/hectorj2f/almighty-core@7c19027ff2a796f889b4995148e8b77672addcba/-/blob/path/path.go#L87:1-88:1)
Can we keep one and use the same ?

The purpose of `path.Path` was to eliminate duplicates.. Few methods are not exported but we can export them as needed in future.